deploy/docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM alpine:3.5
+
+LABEL maintainer="adundovi@gmail.com"
+LABEL description="Prepares a docker image \
+of pump.io server, http://pump.io/."
+
+ENV LC_ALL en_US.UTF-8
+
+RUN apk add --no-cache \
+  nodejs \
+  git \
+  python \
+  make \
+  g++ \
+  graphicsmagick
+RUN npm install -g databank-mongodb
+RUN npm install -g pump.io
```

Comment:
@adundovi the images we'd upload to Docker Hub would be built from tagged releases. Building the image from the cloned source tree is good for a couple reasons:

1. If you check out a tag and build the Docker image, you'll always get an image with the code at that tag (instead of whatever's the latest version on npm)
2. We can test that the Docker image works in CI
3. It'll work for development, for contributors who already have Docker set up and find it convenient to use it to manage the environment